pkg/security/tests/module_tester.go

```diff
@@ -316,6 +316,21 @@ func which(tb testing.TB, name string) string {
 	return executable
 }
 
+//nolint:deadcode,unused
+// whichNotFatal returns an error instead of fatal
+func whichNotFatal(tb testing.TB, name string) (string, error) {
```

Comment:
What about making the `which` function call the `whichNotFatal` function and make the t.Fatalf ?

Reply:
good catch!